### ISSUE TITLE
Can't turn off tooltip in mobile view

### ISSUE THREAD

**Issue by kynotvok** [liprec/azurebi-docs#65]
Hi. Turning off tooltip doesn't work for mobile optimized reports which makes it almost impossible to work with slicer in mobile view. Is it a bug?

⚠ Do not edit this section. It is required for docs.jppp.org ➟ GitHub issue linking.

* document_id=487cd175-be13-40df-a80e-aa1c7f531eb1
* document_url=https://azurebi-docs.jppp.org/powerbi-visuals/hierarchy-slicer.html

**Comment by liprec** [comment]
Did you turn off the tooltips in the formatting pane?
If yes and they still appear, then it related to the Power BI Mobile App itself and not the visual as the visual doesn't do if it 'running' in mobile view.

-JP

**Comment by kynotvok** [comment]
Yes, I turned it off.
Ok, thanks.

**Comment by kynotvok** [comment]
Hi again!
I've posted this issue on power bi portal and got the following answer:  

"The tooltip implementation in this visual is wrong.
Since this is a custom visual, we contacted the visual owner and asked the developer to fix the implementation.
Once the fix will be submitted to the custom visual team, a certification process will start, which might take some time.
While this happens, the customer can contact the visual owner directly and ask for a ‘private’ instance of this visual with a fix.
The visual’s contact info can be found in the ‘about dialog’ on the visual."

Did you recevice any feedback from them? 

Link to the issue: https://community.powerbi.com/t5/Issues/Can-t-turn-off-tooltips-in-phone-layout-for-Hierarchy-slicer/idi-p/838789/page/2#comments
